--- a/src/Admin/Appearance.php
+++ b/src/Admin/Appearance.php
@@ -23,6 +23,8 @@
  */
 class Appearance
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Comments.php
+++ b/src/Admin/Comments.php
@@ -23,6 +23,8 @@
  */
 class Comments
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Dashboard.php
+++ b/src/Admin/Dashboard.php
@@ -23,6 +23,8 @@
  */
 class Dashboard
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Media.php
+++ b/src/Admin/Media.php
@@ -23,6 +23,8 @@
  */
 class Media
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Pages.php
+++ b/src/Admin/Pages.php
@@ -23,6 +23,8 @@
  */
 class Pages
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Plugins.php
+++ b/src/Admin/Plugins.php
@@ -23,6 +23,8 @@
  */
 class Plugins
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Posts.php
+++ b/src/Admin/Posts.php
@@ -23,6 +23,8 @@
  */
 class Posts
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Settings.php
+++ b/src/Admin/Settings.php
@@ -23,6 +23,8 @@
  */
 class Settings
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Support/BlockEditor.php
+++ b/src/Admin/Support/BlockEditor.php
@@ -19,6 +19,8 @@
  */
 class BlockEditor
 {
+    public $editor;
+
     protected $config = [];
 
     /**
--- a/src/Admin/Support/Router.php
+++ b/src/Admin/Support/Router.php
@@ -17,6 +17,8 @@
  */
 class Router
 {
+    public $key;
+
     protected $routes;
     protected $route = false;
 
--- a/src/Admin/Tools.php
+++ b/src/Admin/Tools.php
@@ -23,6 +23,8 @@
  */
 class Tools
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Admin/Users.php
+++ b/src/Admin/Users.php
@@ -24,6 +24,8 @@
  */
 class Users
 {
+    public $config;
+
     /**
      * Initialize
      *
--- a/src/Application/Media/Sizes.php
+++ b/src/Application/Media/Sizes.php
@@ -26,6 +26,8 @@
  */
 class Sizes
 {
+    public $api;
+
     protected $config;
     protected $sizes;
     protected $defaults;
--- a/src/Application/Media/Uploads.php
+++ b/src/Application/Media/Uploads.php
@@ -22,6 +22,8 @@
  */
 class Uploads
 {
+    public $api;
+
     protected $config;
 
     /**
--- a/src/Application/Permalinks.php
+++ b/src/Application/Permalinks.php
@@ -25,6 +25,8 @@
  */
 class Permalinks
 {
+    public $api;
+
     protected $config;
 
     /**
--- a/src/Application/Privacy.php
+++ b/src/Application/Privacy.php
@@ -21,6 +21,8 @@
  */
 class Privacy
 {
+    public $api;
+
     protected $config;
 
     /**
--- a/src/Support/Str.php
+++ b/src/Support/Str.php
@@ -37,7 +37,7 @@
      */
     public static function explode($delimiter, $value, $limit = PHP_INT_MAX)
     {
-        return Arr::collect(explode($delimiter, $value, $limit));
+        return Arr::collect(explode($delimiter, (string) $value, $limit));
     }
 
     /**
--- a/src/UserInterface.php
+++ b/src/UserInterface.php
@@ -20,6 +20,9 @@
  */
 class UserInterface
 {
+    public $routeExport;
+    public $routeImport;
+
     /**
      * Initialize
      */
--- a/vendor/illuminate/collections/Arr.php
+++ b/vendor/illuminate/collections/Arr.php
@@ -153,7 +153,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public static function first($array, callable $callback = null, $default = null)
+    public static function first($array, ?callable $callback = null, $default = null)
     {
         if (\is_null($callback)) {
             if (empty($array)) {
@@ -178,7 +178,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public static function last($array, callable $callback = null, $default = null)
+    public static function last($array, ?callable $callback = null, $default = null)
     {
         if (\is_null($callback)) {
             return empty($array) ? value($default) : \end($array);
--- a/vendor/illuminate/collections/Collection.php
+++ b/vendor/illuminate/collections/Collection.php
@@ -300,7 +300,7 @@
      * @param  callable|null  $callback
      * @return static
      */
-    public function filter(callable $callback = null)
+    public function filter(?callable $callback = null)
     {
         if ($callback) {
             return new static(Arr::where($this->items, $callback));
@@ -314,7 +314,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public function first(callable $callback = null, $default = null)
+    public function first(?callable $callback = null, $default = null)
     {
         return Arr::first($this->items, $callback, $default);
     }
@@ -559,7 +559,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public function last(callable $callback = null, $default = null)
+    public function last(?callable $callback = null, $default = null)
     {
         return Arr::last($this->items, $callback, $default);
     }
--- a/vendor/illuminate/collections/Enumerable.php
+++ b/vendor/illuminate/collections/Enumerable.php
@@ -23,7 +23,7 @@
      * @param  callable|null  $callback
      * @return static
      */
-    public static function times($number, callable $callback = null);
+    public static function times($number, ?callable $callback = null);
     /**
      * Create a collection with the given range.
      *
@@ -234,7 +234,7 @@
      * @param  callable|null  $callback
      * @return static
      */
-    public function filter(callable $callback = null);
+    public function filter(?callable $callback = null);
     /**
      * Apply the callback if the value is truthy.
      *
@@ -243,7 +243,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function when($value, callable $callback, callable $default = null);
+    public function when($value, callable $callback, ?callable $default = null);
     /**
      * Apply the callback if the collection is empty.
      *
@@ -251,7 +251,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function whenEmpty(callable $callback, callable $default = null);
+    public function whenEmpty(callable $callback, ?callable $default = null);
     /**
      * Apply the callback if the collection is not empty.
      *
@@ -259,7 +259,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function whenNotEmpty(callable $callback, callable $default = null);
+    public function whenNotEmpty(callable $callback, ?callable $default = null);
     /**
      * Apply the callback if the value is falsy.
      *
@@ -268,7 +268,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function unless($value, callable $callback, callable $default = null);
+    public function unless($value, callable $callback, ?callable $default = null);
     /**
      * Apply the callback unless the collection is empty.
      *
@@ -276,7 +276,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function unlessEmpty(callable $callback, callable $default = null);
+    public function unlessEmpty(callable $callback, ?callable $default = null);
     /**
      * Apply the callback unless the collection is not empty.
      *
@@ -284,7 +284,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function unlessNotEmpty(callable $callback, callable $default = null);
+    public function unlessNotEmpty(callable $callback, ?callable $default = null);
     /**
      * Filter items by the given key value pair.
      *
@@ -380,7 +380,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public function first(callable $callback = null, $default = null);
+    public function first(?callable $callback = null, $default = null);
     /**
      * Get the first item by the given key value pair.
      *
@@ -488,7 +488,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public function last(callable $callback = null, $default = null);
+    public function last(?callable $callback = null, $default = null);
     /**
      * Run a map over each of the items.
      *
--- a/vendor/illuminate/collections/LazyCollection.php
+++ b/vendor/illuminate/collections/LazyCollection.php
@@ -320,7 +320,7 @@
      * @param  callable|null  $callback
      * @return static
      */
-    public function filter(callable $callback = null)
+    public function filter(?callable $callback = null)
     {
         if (\is_null($callback)) {
             $callback = function ($value) {
@@ -342,7 +342,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public function first(callable $callback = null, $default = null)
+    public function first(?callable $callback = null, $default = null)
     {
         $iterator = $this->getIterator();
         if (\is_null($callback)) {
@@ -554,7 +554,7 @@
      * @param  mixed  $default
      * @return mixed
      */
-    public function last(callable $callback = null, $default = null)
+    public function last(?callable $callback = null, $default = null)
     {
         $needle = $placeholder = new stdClass();
         foreach ($this as $key => $value) {
--- a/vendor/illuminate/collections/Traits/EnumeratesValues.php
+++ b/vendor/illuminate/collections/Traits/EnumeratesValues.php
@@ -105,7 +105,7 @@
      * @param  callable|null  $callback
      * @return static
      */
-    public static function times($number, callable $callback = null)
+    public static function times($number, ?callable $callback = null)
     {
         if ($number < 1) {
             return new static();
@@ -387,7 +387,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function when($value, callable $callback = null, callable $default = null)
+    public function when($value, ?callable $callback = null, ?callable $default = null)
     {
         if (!$callback) {
             return new HigherOrderWhenProxy($this, $value);
@@ -406,7 +406,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function whenEmpty(callable $callback, callable $default = null)
+    public function whenEmpty(callable $callback, ?callable $default = null)
     {
         return $this->when($this->isEmpty(), $callback, $default);
     }
@@ -417,7 +417,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function whenNotEmpty(callable $callback, callable $default = null)
+    public function whenNotEmpty(callable $callback, ?callable $default = null)
     {
         return $this->when($this->isNotEmpty(), $callback, $default);
     }
@@ -429,7 +429,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function unless($value, callable $callback, callable $default = null)
+    public function unless($value, callable $callback, ?callable $default = null)
     {
         return $this->when(!$value, $callback, $default);
     }
@@ -440,7 +440,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function unlessEmpty(callable $callback, callable $default = null)
+    public function unlessEmpty(callable $callback, ?callable $default = null)
     {
         return $this->whenNotEmpty($callback, $default);
     }
@@ -451,7 +451,7 @@
      * @param  callable|null  $default
      * @return static|mixed
      */
-    public function unlessNotEmpty(callable $callback, callable $default = null)
+    public function unlessNotEmpty(callable $callback, ?callable $default = null)
     {
         return $this->whenEmpty($callback, $default);
     }
